rosco-core/src/main/groovy/com/netflix/spinnaker/rosco/providers/CloudProviderBakeHandler.groovy

```diff
@@ -120,6 +120,8 @@ abstract class CloudProviderBakeHandler {
 
     parameterMap.configDir = configDir
 
+    parameterMap.put("upgrade", bakeRequest.upgrade)
```

Comment:
Please handle the setting of this new parameter the same way as `bakeRequest.build_host` just above.